src/plugins/mpi/pmix/pmixp_client_v2.c

```diff
@@ -88,12 +88,24 @@ static pmix_status_t _abort_fn(const pmix_proc_t *proc, void *server_object,
 {
 	/* Just kill this stepid for now. Think what we can do for FT here? */
 	PMIXP_DEBUG("called: status = %d, msg = %s", status, msg);
-	slurm_kill_job_step(pmixp_info_jobid(), pmixp_info_stepid(), SIGKILL);
 
-	if (NULL != cbfunc) {
-		cbfunc(PMIX_SUCCESS, cbdata);
+	slurm_addr_t abort_server;
+	abort_server.sin_family = AF_INET;
+	abort_server.sin_port = pmixp_info_abort_agent_port();
+	abort_server.sin_addr.s_addr = inet_addr(pmixp_info_srun_ip());
+
+	int client_sock;
+	if((client_sock = slurm_open_msg_conn(&abort_server)) < 0){
+		PMIXP_ERROR("Error create and conn client socket: %s", strerror(errno));
+		return SLURM_ERROR;
 	}
-	return PMIX_SUCCESS;
+
+	slurm_write_stream(client_sock, &status, sizeof(status));
```

Comment:
```suggestion
	slurm_write_stream(client_sock, &status_net, sizeof(status_net));
```

Reply:
Receive response from the srun that data is successfuly in place
slurm_read_stream(client_sock, &status, sizeof(status));